sssd_test_framework/utils/adcli.py

```diff
@@ -0,0 +1,92 @@
+"""Perform actions on Active Directory."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.conn import ProcessResult
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "ADCLIUtils",
+]
+
+
+class ADCLI(MultihostUtility[MultihostHost]):
+    """
+    Call commands from adcli
```

Comment:
Make a note that this utility does not revert changes.

Given the use case, I think it is fine to rely on reverting the AD host in the topology controller, but it needs to be documented.